appinsights/client.go

```diff
@@ -111,8 +119,8 @@ func (tc *telemetryClient) TrackTraceTelemetry(trace *TraceTelemetry) {
 	tc.Track(item)
 }
 
-func (tc *telemetryClient) TrackRequest(name string, timestamp time.Time, duration time.Duration, responseCode string, success bool) {
-	item := NewRequestTelemetry(name, timestamp, duration, responseCode, success)
+func (tc *telemetryClient) TrackRequest(name, method, url string, timestamp time.Time, duration time.Duration, responseCode string, success bool) {
+	item := NewRequestTelemetry(name, method, url, timestamp, duration, responseCode, success)
```

Comment:
Seeing that all this does is generate `RequestTelemetry`, I'm going to double my suggestion to just take `RequestTelemetry` as an argument here